### ISSUE TITLE
[Bug]: BVT failed

### ISSUE THREAD

**Issue by zhangxu19830126** [matrixorigin/matrixone#10165]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

https://github.com/matrixorigin/matrixone/actions/runs/5328784493?pr=10129,

mo hung and can not get a valid connection

**Comment by XuPeng-SH** [comment]
#10168

**Comment by XuPeng-SH** [comment]
fixed by #10168
